app/views/admin/resources/index.html.haml

```diff
@@ -20,17 +22,16 @@
                 = link_to(admin_conference_resource_path(@conference.short_title, resource.id)) do
                   = resource.name
                 %td
-                  = quantity_left_of(resource)
+                  %span{ id: "update_actions_#{resource.id}" }
+                    = render partial: 'update_actions', locals: { conference: @conference, resource: resource }
                 %td
-                  = resource.used
+                  %span{ id: "resource_used_#{resource.id}" }
+                    = resource.used
                 %td
                   .btn-group
                     = link_to 'Edit', edit_admin_conference_resource_path(@conference.short_title, resource.id),
                       method: :get, class: 'btn btn-primary', disabled: !(can? :update, resource)
                     = link_to 'Delete', admin_conference_resource_path(@conference.short_title, resource.id),
                       method: :delete, class: 'btn btn-danger', disabled: !(can? :destroy, resource),
                       data: { confirm: "Do you really want to delete #{resource.name}?" }
-.row
```

Comment:
I asked for your opinion on this in the comment [link](https://github.com/openSUSE/osem/pull/1301#issuecomment-292151739). I will repeat what I proposed: 

- Unable to find it as there are paging 's next and previous action which occupies large space
- It is convenient to have important actions at the top rather than bottom e.g When we use google keep we have input to add a task at the top instead of the bottom.